run_modal_docker.py

```
@@ -0,0 +1,244 @@
+        try:
+            while process.poll() is None:
+                # プロセスのログを定期的に出力
+                try:
+                    stdout_line = process.stdout.readline()
+                    if stdout_line:
+                        print(f"[WebUI] {stdout_line.decode().strip()}")
+                    
+                    stderr_line = process.stderr.readline()
+                    if stderr_line:
+                        print(f"[WebUI ERROR] {stderr_line.decode().strip()}")
+                except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)